CHANGELOG.md

```diff
@@ -19,6 +19,10 @@
     - `HORIZONTAL ENCODER ◀︎▶︎` + `PLAY` is changed to `CROSS SCREEN` + `PLAY`
   - `Menu Highlighting (HIGH)` changes how menu highlighting is rendered on `OLED` displays by drawing a vertical bar `|` on the left edge of the display beside the selected menu item instead of highlighting the area of the selected menu item by inverting the text.
 
+#### <ins>Horizontal Menus</ins>
+- Following items display in horizontal menus on OLED, with multiple values visible and editable at the same time. Hold `SHIFT` and turn `SELECT` to edit them (horizontal menus can be disabled via Community Features):
```

Comment:
"Following items" is confusing wording to me. Could we update this to be more clear?